### DISCUSSION TITLE
Creality Filament sensor

### DISCUSSION THREAD

**Discussion by feiticeir0** [Jyers/Marlin#1858] [Q&A] [answered]
Hi all ! 
I've bought the Creality filament sensor for my Ender 3 v2. I'm now waiting on the CR-touch 

I'm no stranger to compiling Marlin. 
My question is: 
In the latest release, does the BL-Touch version will work with the CR-Touch ? 

And, I've already downloaded the source code. I need to enable the filament sensor. 

 I can see two folders - E3V2 templates and Jyers config. What's the difference ? 

I'm sorry if these are noob questions, but since I never compiled Marlin for this printer, I'm a little apprehensive. 

Thank you

**Comment by BlueWings172** [comment, marked as answer]
@feiticeir0 

> does the BL-Touch version will work with the CR-Touch ?

Yes, CR-Touch should work. I haven't tried myself but I have seen people using it with this firmware.

> I can see two folders - E3V2 templates and Jyers config. What's the difference ?

Jyers config is what Jyers uses for his own printer. Don't use it, use the configuration in the E3V2 templates folder. in the latest version, the folders have been renamed to clarify that. You are probably haven't downloaded the latest version, it is here #1694

**Reply by feiticeir0** [reply]
Thank you @BlueWings172  for the clarification. Will update my sources . 
Have a nice weekend.
